safe_core/src/connection_manager/connection_group.rs

```diff
@@ -346,7 +346,7 @@ impl Connected {
             .remove(&msg_id)
             .map(|(sender, count)| {
                 let count = count - 1;
-                dbg!("Response no: {}", count);
+                //                dbg!("Response no: {}", count);
```

Comment:
I guess this isnt needed. 

(but also, i've no seen that, what is it vs `trace!` eg?

Reply:
Yea, we don't need this change, will revert now. 

`dbg!()` is Rust's debug macro. It works if `Debug` trait is implemented on the given entity(works much like `println!()` but with more info). 
`trace!` is a logging utility which can be in release versions whereas `dbg!()` is used only for debugging & we remove it once done developing.